features/profile/view/NewHostRequest.tsx

```diff
@@ -176,6 +177,7 @@ export default function NewHostRequest({
               id="from-date"
               label={t("profile:request_form.arrival_date")}
               name="fromDate"
+              defaultValue={null}
```

Comment:
I think it should be undefined, not null - I get this ts warning:
`Type 'null' is not assignable to type 'Dayjs | undefined'.ts(2322)
Datepicker.tsx(22, 3): The expected type comes from property 'defaultValue' which is declared here on type 'IntrinsicAttributes & DatepickerProps'`
Think it makes sense since it wasn't actively set to null, just hasn't been defined yet

Reply:
hmm, I'm not getting this error (not Firefox neither Chrome)... are you copying the changes to your local branch? (instead of pulling the branch?)

Reply:
updated now :) (finally the bug was caused by the minDate prop)

Reply:
hmm sorry, i'm not sure why but when i look at it with undefined as the default value, it populates the field with today's date 😅 maybe null was the way to go!

Reply:
hmm, that's weird.. are you using the same branch as mine? (doesn't happen to me though)